modules/ROOT/pages/migrate-from-an-earlier-version-of-bonita-bpm.adoc

```diff
@@ -224,138 +223,131 @@ xref:back-up-bonita-bpm-platform.adoc[Back up your platform] and database in cas
 | `bonita-migration-distrib-sp` (Linux) or `bonita-migration-distrib-sp.bat` (Windows)
 |===
 
- ** Starting from version 2.44.1, an additional script called `check-migration-dryrun` is present in the same folder. This script only run checks the migration would without actually migrating. This is equivalent to running the migration script with a `--verify` option.
-. The script detects the current version of Bonita, and displays a list of the versions that you can migrate to. Specify the
- version you require.
- The script starts the migration.
-. As the script runs, it displays messages indicating progress. After each migration step, you are asked to confirm whether to
- proceed to the next step. You can pause the migration by answering `no`.
- To suppress the confirmation questions, so that the migration can run unattended, set the ` (-Dauto.accept=true)` system
+ ** Starting from Bonita Migration Tool 2.44.1, an additional script called `check-migration-dryrun` is present in the same folder. This script only runs the checks the real update would run, without actually migrating the elements. This is equivalent to running the migration script with a `--verify` option.
```

Comment:
runs the checks the real update would run, without actually migrating the elements. => runs the checks, the real update would run without actually migrating the elements.